util/cmdtest/cmds.go

```diff
@@ -50,7 +51,7 @@ func StoppableCtx() (context.Context, func()) {
 }
 
 // StartDummy starts dummy analyzer with context and optional arguments
-func StartDummy(ctx context.Context, args ...string) io.Reader {
+func StartDummy(ctx context.Context, require *require.Assertions, args ...string) io.Reader {
```

Comment:
maybe make this functions part of `IntegrationSuite` too? Then we won't need to pass `require.Assertions` everywhere

Reply:
We also use these functions in `cmd/sdk-test/`, I don't see a clean way to make it part of both packages :thinking:  Any suggestion?

Reply:
define `IntegrationSuite` struct in `cmdtest` package?

Like
```go
type IntegrationSuite struct {
 	suite.Suite
 	ctx  context.Context
 	stop func()
}
```

Then both `ServerIntegrationSuite` and `BblfshIntegrationSuite` can inherit it.

Just make sure it's not overengineering 😅

Reply:
done in bf58993